### DISCUSSION TITLE
npx prisma init > Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/06fc58a368dc7be9fbbbe894adf8d445d208c284/windows/query_engine.dll.node.gz.sha256 failed, reason: getaddrinfo EAI_AGAIN binaries.prisma.sh

### DISCUSSION THREAD

**Discussion by donafqiu** [prisma/prisma#25318] [Q&A]
### Question

```
    "@types/node": "^22.7.4",
    "prisma": "^5.20.0",
    "ts-node": "^10.9.2",
    "typescript": "^5.6.2"
```

[https://www.prisma.io/docs/getting-started/quickstart](url)

When I execute `npx prisma migrate dev --name init`
Reported a mistake
> Downloading Prisma engines for Node-API for windows [                    ] 0%Error:  request to  https://binaries.prisma.sh/all_commits/06fc58a368dc7be9fbbbe894adf8d445d208c284/windows/query_engine.dll.node.gz.sha256  failed, reason: getaddrinfo EAI_AGAIN binaries.prisma.sh

But It works fine on another computer

Has anyone experienced similar problems?

Could you help me with that?

Thank you very much

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @donafqiu 

This might be a network-related issue when downloading the Prisma engines on your current machine. 

Can you try to manually download the file by opening the URL in your browser. This can help determine if the issue is with the network or the Prisma CLI:

```
https://binaries.prisma.sh/all_commits/06fc58a368dc7be9fbbbe894adf8d445d208c284/windows/query_engine.dll.node.gz.sha256
```

If the file downloads successfully, the network is not the issue. Sometimes, using a VPN can help bypass network restrictions that might be causing the issue.

**Reply by donafqiu** [reply]
![image](https://github.com/user-attachments/assets/a26d3a1b-ab44-42b0-807f-bcf6bc3c48f4)
![image](https://github.com/user-attachments/assets/5e574a33-82d9-49b8-a595-d773fcc4d809)

Are you sure that is network-related issue?
But I open the site, it also error,too...
I asked other friends to vistit this site, also 404...

**Reply by ludralph** [reply]
Can you please update to the latest Prisma version 6.2.1 and try to see if this issue is resolved. In case it's not resolved, please open  an [issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
